src/helpers/contexts/GlobalNodeState.tsx

```diff
@@ -76,6 +76,8 @@ interface Global {
     setIteratorPercent: (id: string, percent: number) => void;
     setHoveredNode: SetState<string | null | undefined>;
     setZoom: SetState<number>;
+    isBackendKilled: boolean;
```

Comment:
This has nothing to do with current global *node* state.

Why not move this into the execution context?

Reply:
Good point